docs/develop/persisting-add-in-state-and-settings.md

```diff
@@ -1,31 +1,147 @@
-    > [!NOTE]
-    > Some browsers or the user's browser settings may block browser-based storage techniques. You should test for availability as documented in [Using the Web Storage API](https://developer.mozilla.org/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API).
-    >
-    > Starting in Version 115 of Chromium-based browsers, such as Chrome and Edge, [storage partitioning](https://developer.chrome.com/docs/privacy-sandbox/storage-partitioning/) is being tested to prevent specific side-channel cross-site tracking (see also [Microsoft Edge browser policies](/deployedge/microsoft-edge-policies#defaultthirdpartystoragepartitioningsetting)). This means that data stored by storage APIs, such as local storage, are only available to contexts with the same origin and the same top-level site. To work around this, in your browser, go to **chrome://flags** or **edge://flags**, then set the **Experimental third-party storage partitioning (#third-party-storage-partitioning)** flag to **Disabled**.
+### How to save settings in an Outlook add-in
 
-This article focuses on how to use the Office JavaScript API to persist add-in state to the current document. It's recommended that you use the application-specific object if it's available for your selected Office client instead of the Common Office JavaScript version. If you need to persist state across documents, such as tracking user preferences across any documents they open, you'll need to use a different approach. For example, you could use [SSO](use-sso-to-get-office-signed-in-user-token.md) to obtain the user identity, and then save the user ID and their settings to an online database.
+For information about how to save settings in an Outlook add-in, see [Get and set add-in metadata for an Outlook add-in](../outlook/metadata-for-an-outlook-add-in.md) and [Get and set internet headers on a message in an Outlook add-in](../outlook/internet-headers.md).
 
-## Persist add-in state and settings with the Office JavaScript API
+## Common API settings and persistence
 
-The Office JavaScript API provides objects, such as [Settings](/javascript/api/office/office.settings), [RoamingSettings](/javascript/api/outlook/office.roamingsettings), and [CustomProperties](/javascript/api/outlook/office.customproperties), to save add-in state across sessions as described in the following table. In all cases, the saved settings values are associated with the [Id](/javascript/api/manifest/id) of the add-in that created them.
+The [Common APIs](understanding-the-javascript-api-for-office.md#api-models) provide objects to save add-in state across sessions, as described in the following table. In all cases, the saved settings values are associated with the [Id](/javascript/api/manifest/id) of the add-in that created them.
 
 |Object|Add-in type support|Storage location|Office application support|
```

Comment:
I think we can remove this table. Aside from Settings and CustomXmlParts, the rest of the APIs in the table are now described in the application-specific section.

If we remove the table, a section describing Excel.CustomProperty and Word.CustomProperty will need to be added to the application-specific section.

Reply:
Good call. I'll pull that information from the Excel article that talks about custom properties.